frontend/src/modules/Servicemap/ServiceMap.tsx

```diff
@@ -150,7 +150,9 @@ const mapStateToProps = (
 	};
 };
 
-export default connect(mapStateToProps, {
-	getServiceMapItems: getServiceMapItems,
-	getDetailedServiceMapItems: getDetailedServiceMapItems,
-})(ServiceMap);
+export default withRouter(
```

Comment:
Whats the use of adding `withRouter`

Reply:
Loadable expect `ComponentType` but the connect returns `ConnectedComponent` that's the reason I wrapped with `withRouter` which return `ComponentType`